SayTheirNames/Source/View/Home/Person/PersonCell.swift

```diff
@@ -125,7 +125,7 @@ final class PersonCell: UICollectionViewCell {
             bookmarkButton.topAnchor.constraint(equalTo: nameLabel.topAnchor),
             bookmarkButton.trailingAnchor.constraint(equalTo: labelsAndButtonContainer.trailingAnchor),
 
-            profileImageView.heightAnchor.constraint(lessThanOrEqualTo: containerStack.heightAnchor, multiplier: 0.8),
+            profileImageView.heightAnchor.constraint(equalTo: containerStack.heightAnchor, multiplier: 0.8),
 
```

Comment:
Just make sure this didn't break stuff on the various screen sizes (changing <= to ==)

Reply:
Not mine, silly me did a fetch and pull after my changes